File: Programming Basics/Conditional Statements - Exercises/08.LunchBreak.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string seriesName = Console.ReadLine();
        double episodeLength = double.Parse(Console.ReadLine());
        double breakLength = double.Parse(Console.ReadLine());

        double lunchTime = breakLength / 8;
        double relaxTime = breakLength / 4;

        double timeLeft = breakLength - (lunchTime + relaxTime);

        if (timeLeft >= episodeLength)
        {
            Console.WriteLine($"You have enough time to watch {seriesName} and left with {Math.Ceiling(timeLeft - episodeLength)} minutes free time.");
        }
        else
        {
            Console.WriteLine($"You don't have enough time to watch {seriesName}, you need {Math.Ceiling(episodeLength - timeLeft)} more minutes.");
        }
    }
}

File: Programming Basics - Python/Conditional Statements - Exercises/08.LunchBreak.py
from math import ceil

series_name = input()
length = int(input())
break_length = int(input())

actual_time = break_length - ((break_length / 8) + (break_length / 4))

if actual_time >= length:
    print(f'You have enough time to watch {series_name} and left with {ceil(actual_time - length):.0f} minutes free time.')

else:
    print(f'You don\'t have enough time to watch {series_name}, you need {ceil(length - actual_time):.0f} more minutes.')